### PR TITLE
aws s3: add page

### PR DESCRIPTION
**Pull request by dac09**
Adds a few common aws-s3 cli commands.

Since the aws cli is quite large, I thought it would be useful to enter s3 separately. I'm assuming what we can do is `tldr 'aws s3'`.

### CODE REVIEW

**Comment by waldyrious** [comment]
@dac09 yes, splitting large commands into pages for each subcommand is what we do already (see git). Feel free to send the remaining ones :)

As for the contents of this page, I'll leave it to someone who has used AWS to comment.

**Comment by dac09** [comment]
@waldyrious Is there a way I could try this locally?

**Comment by agnivade** [comment]
Yes, `tldr aws s3` would work(atleast in the node client). Mind that its without the quotes.

> Is there a way I could try this locally?

Not sure I understood you. You want to create multiple pages and test them ? That can always be done with `npm test`

> Also is the file name okay or should it be aws-s3.md?

Yes, it should be aws-s3.md

**Comment by dac09** [comment]
Rebased and pushed the file name change :).

I meant - is there a way I can point to local documents with tldr? So while it isn't merged, is there a way I could still use it with tldr on the command line? (node client)

**Comment by agnivade** [comment]
Yes - `tldr --render {{path/to/local_file}}`

**Comment by waldyrious** [comment]
Any chance you could shorten the page by a couple examples?

**Comment by dac09** [comment]
Yes - sure. I'll also make a couple of small changes to make it render better in tldr - the `{{sourcebucket}}{{path/to/file}}` doesn't highlight properly

**Comment by dac09** [comment]
Updated! Here's what it looks like
screenshot 2016-09-29 10 29 29

**Comment by waldyrious** [comment]
very nice! LGTM :)

File: pages/common/aws-s3.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+`aws s3 sync {{path/to/files}} s3://{{bucket_name}} --exclude {{path/to/file}} --exclude {{path/to/folder}}/*`
+
+- Copy from one bucket to another recursively:
+
+`aws s3 cp s3://{{source_bucket}}/{{path/to/file}} s3://{{destination_bucket}} --recursive`
+
+- Remove file from bucket:
+
+`aws s3 rm s3://{{source_bucket}}/{{path/to/file}}`
```
Comment (agnivade) [inline pages/common/aws-s3.md]: source_bucket -> bucket. As source and destination does not have a meaning here.

**Comment by agnivade** [comment]
Hey @dac09 , any updates on this ?

**Comment by dac09** [comment]
@agnivade updated - thanks for the reminder 👍

**Comment by agnivade** [comment]
LGTM

**Pull request merged**